mllib/src/main/scala/org/apache/spark/mllib/stat/MultivariateOnlineSummarizer.scala

```diff
@@ -233,7 +233,7 @@ class MultivariateOnlineSummarizer extends MultivariateStatisticalSummary with S
   override def numNonzeros: Vector = {
     require(totalWeightSum > 0, s"Nothing has been added to this summarizer.")
```

Comment:
yeah, here it should use `totalCnt`, I update it.
and I fix the testcase, now the existing testcase already cover the method.
thanks!